<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDEMDOMProcess</class>
 <widget class="QDialog" name="DialogDEMDOMProcess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSource">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>391</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>输入文件路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDest">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>391</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSource">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>120</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>输出文件路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDest">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>180</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxLeftTopX">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxLeftTopY">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxRightBottomY">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>280</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxRightBottomX">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>230</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxSampleCoef">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>左上角X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>左上角Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>右下角X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>右下角Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>重采样系数</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonPixelBased">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>240</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>按像素坐标</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonGeoBased">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>270</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>按地理坐标</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>指定波段</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxBandNum">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>280</y>
     <width>59</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogDEMDOMProcess</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogDEMDOMProcess</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
